demo/icons/icon-size-override.js

```diff
@@ -0,0 +1,23 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+
+class IconSizeOverride extends LitElement {
+
+	static get styles() {
+		return css`
+			:host {
+				display: inline-block;
+			}
+			::slotted(d2l-icon) {
+				--d2l-icon-height: 100px;
+				--d2l-icon-width: 100px;
```

Comment:
So this doesn't work in IE11 only... Firefox, Safari, Edge and Chrome it's fine. Uh... any ideas?

Reply:
Interesting.  I wonder if shadyCSS is scoping those variables to the IconSizeOverride local DOM scope?  Or, does it make any difference if those assignments are done inside of `:host` instead?

Reply:
I tried that as well, also doesn't work. When I inspect the CSS that's getting inserted into the `<head>`, there's nothing there.